README.md

```diff
@@ -44,6 +46,10 @@ compile 'io.opencensus:opencensus-proto:0.1.0'
 [travis-url]: https://travis-ci.org/census-instrumentation/opencensus-proto
 [maven-image]: https://maven-badges.herokuapp.com/maven-central/io.opencensus/opencensus-proto/badge.svg
 [maven-url]: https://maven-badges.herokuapp.com/maven-central/io.opencensus/opencensus-proto
+[gitter-image]: https://badges.gitter.im/census-instrumentation/lobby.svg
+[gitter-url]: https://gitter.im/census-instrumentation/lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
```

Comment:
I think this URL can just be `https://gitter.im/census-instrumentation/lobby` without the query string `?*`

Reply:
SG, done. Also added GoDoc.